Source/BestProjectEver/Public/Weapons/BPE_Weapon.h

```diff
@@ -77,7 +78,7 @@ class BESTPROJECTEVER_API ABPE_Weapon : public AActor
 	TObjectPtr<ABPE_BaseCharacter> OwnerCharacter;
 	
 	/** Camera can be inverted or normal */
-    UPROPERTY(EditDefaultsOnly, BlueprintReadOnly, Category = "Aiming")
+    UPROPERTY(EditAnywhere, BlueprintReadOnly, Category = "Aiming")
     uint8 bIsLookInverted : 1;
```

Comment:
Didn't notice before but why do you need this setting per weapon. Shouldn't this be an overall user setting?

Reply:
Yesss I pasted it by accident there